src/hooks/useOrder.ts

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect, useCallback } from "react";
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "../modules/Store";
+import { getOrders } from "../modules/Orders";
+
+function useOrder() {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  const [totalOrderPrice, setTotalOrderPrice] = useState(0);
+
+  const orders = useSelector((state: RootState) => state.orders.ordersList);
+  const showModal = useSelector((state: RootState) => state.modal.showModal);
+
+  const orderDetails = orders
+    .map((order) => {
+      return order.orderDetails;
+    })
+    .reduce((previous, current) => {
+      return [...previous, ...current];
+    }, []);
```

Comment:
```suggestion
    .map((order) => order.orderDetails)
    .reduce((previous, current) => [...previous, ...current], []);
```

return 문을 축약해서 사용하는건 어떨까용?

Reply:
아래 작성하신 `handleOrderTotal`는 일부 메서드의 return이 축약되어 있어서 전체적인 컨벤션이 통일 되면 코드의 가독성이 더 올라갈거 같아요!